flytekit/common/launch_plan.py

```diff
@@ -88,6 +92,27 @@ def register(self, project, domain, name, version):
         self._id = id_to_register
         return str(self.id)
 
+    @_exception_scopes.system_entry_point
+    def fast_register(self, project, domain, name, already_uploaded_digest=None, working_dir=None) -> str:
```

Comment:
I guess this is what Yee is talking about, we probably dont need this to be on the object itself?

Reply:
resolved offline - removed from the RegisterableMixin